README.md

```diff
@@ -24,7 +24,7 @@ Integrate your **DApp** using the [Wallet SDK](https://wallet.fuel.network/docs/
   - [📒 - List user accounts](https://wallet.fuel.network/docs/how-to-use/#list-user-accounts)
   - [✍️ - Signing a message](https://wallet.fuel.network/docs/how-to-use/#signing-a-message)
   - [✍️ - Transfering assets](https://wallet.fuel.network/docs/how-to-use/#transfering-assets)
-  - [📗 SDK API](https://wallet.fuel.network/docs/api/)
+  - [📗 SDK API](https://wallet.fuel.network/docs/dev/getting-started/)
```

Comment:
```suggestion
  - [📗 SDK API](https://wallet.fuel.network/docs/dev/hooks-reference/)
```

@MartinKong1990 thanks for the contribution can you change this to point to https://wallet.fuel.network/docs/dev/hooks-reference/.